### PR TITLE
Advertize PGV & scalapb-validate

### PR DESCRIPTION
**Pull request by bjaglin**

### CODE REVIEW

File: docs/src/main/paradox/apidesign.md [hunk 1]
```diff
@@ -12,3 +12,54 @@ Google in their [empty.proto](https://github.com/protocolbuffers/protobuf/blob/m
...
+  "com.thesamet.scalapb" %% "scalapb-validate-core" % scalapb.validate.compiler.BuildInfo.version % "protobuf"
+
+Compile / PB.targets +=
+  scalapb.validate.gen(FlatPackage) -> (Compile / akkaGrpcCodeGeneratorSettings / target).value
+```
+
+The `validate_at_construction` option can be particularly interesting in a server-side context
+since method implementations will automatically receive pre-validated requests and will not
+be able to return invalid responses. [Rule-based type customization](https://scalapb.github.io/docs/validation/#rule-based-type-customization)
+goes one step further by encoding the rules as types, when applicable.
```
Comment (bjaglin) [inline docs/src/main/paradox/apidesign.md:65]: Both features require the freshly released ScalaPB 0.10.10

**Comment by bjaglin** [comment]
cc @thesamet

**Comment by bjaglin** [comment]
It looks like something went wrong with the release of 1.1.0-M2, so MiMa fails
```
[error] sbt.librarymanagement.ResolveException: Error downloading com.lightbend.akka.grpc:akka-grpc-runtime_2.13:1.1.0-M2
[error]   Not found
[error]   not found: /home/travis/.ivy2/localcom.lightbend.akka.grpc/akka-grpc-runtime_2.13/1.1.0-M2/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/com/lightbend/akka/grpc/akka-grpc-runtime_2.13/1.1.0-M2/akka-grpc-runtime_2.13-1.1.0-M2.pom
[error]   not found: https://oss.sonatype.org/content/repositories/staging/com/lightbend/akka/grpc/akka-grpc-runtime_2.13/1.1.0-M2/akka-grpc-runtime_2.13-1.1.0-M2.pom
[error]   not found: https://dl.bintray.com/akka/snapshots/com/lightbend/akka/grpc/akka-grpc-runtime_2.13/1.1.0-M2/akka-grpc-runtime_2.13-1.1.0-M2.pom
```

**Comment by raboof** [comment]
> It looks like something went wrong with the release of 1.1.0-M2, so MiMa fails

It was delayed by the sonatype troubles earlier (https://status.maven.org/#week), should be on its way to maven central now :crossed_fingers:

**Comment by raboof** [comment]
OK TO TEST

**Review by raboof — APPROVED** [review]
very cool!

**Pull request merged**